lib/cloudinary/railtie.rb

```diff
@@ -3,7 +3,9 @@ class Cloudinary::Railtie < Rails::Railtie
     Dir[File.join(File.dirname(__FILE__),'../tasks/**/*.rake')].each { |f| load f }
   end
   config.after_initialize do |app|
-    ActionView::Base.send :include, CloudinaryHelper
+    ActiveSupport.on_load(:action_view) do
+      ActionView::Base.send :include, CloudinaryHelper

```

Comment:
```suggestion
      ActionView::Base.include(CloudinaryHelper)
```

I maybe miss something, but this should works?